### PR TITLE
[web] 030 train model

### PR DESCRIPTION
**Pull request by elliotmessi**
model in training task;
image selection in model inference

### CODE REVIEW

File: ymir/web/src/locales/modules/task.ts [hunk 1]
```diff
@@ -161,7 +161,12 @@ const task = {
   "task.detail.model.deleted": { cn: '已删除', en: 'Deleted', },
   'task.detail.label.go.platform': { cn: '跳转到标注平台>>', en: 'Go to Label Platform >>' },
   "task.terminate.label.nodata": { cn: '不获取数据终止', en: 'Terminate', },
-  "task.terminate.label.withdata": { cn: '获取结果终止', en: 'Terminate & Fetch Result', },
+  "task.terminate.label.withdata": { cn: '获取数据终止', en: 'Terminate & Fetch Result', },
+  "tip.task.train.model": { cn: '此次训练将会基于选择的模型的基础上继续训练，迭代次数将会累加，目前训练目标需要和模型保持一致', en: 'This training task will base on the model selected, and training target will be ', },
+  "task.train.form.model.label": { cn: '选择模型', en: 'Select Model', },
+  "task.train.form.model.placeholder": { cn: '请选择模型', en: 'Please select a model', },
+  "task.train.target.invalid.inter": { cn: '训练目标必须是训练集和测试集的标签的交集', en: 'Please select a model', },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/task.ts]: 这后面两个是没有更新翻译？

Comment (elliotmessi) [reply ymir/web/src/locales/modules/task.ts]: 忘了翻译，如下可以不：
Training target must be intersection of training sets and test sets
Training target must be as same as model's

Comment (phoenix-xhuang) [reply ymir/web/src/locales/modules/task.ts]: 改成Training target classes，其他没问题

Comment (elliotmessi) [reply ymir/web/src/locales/modules/task.ts]: fixed

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**